spec/predicates/release.md

```diff
@@ -2,54 +2,105 @@
 
 Type URI: https://in-toto.io/attestation/release
 
-Version 0.1
+Version 1
```

Comment:
Any particular reason for the change from v0.1 to v1?

Reply:
At first I didn't understand how to specify a protobuf for v0.1, but I think I figured it out, so I put the version back at 0.1 here and in https://github.com/in-toto/in-toto.io/pull/23.